sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -320,7 +369,7 @@ class wi_element<sycl::ext::oneapi::bfloat16, NumRows, NumCols, Use, Layout,
 
 #if __SYCL_DEVICE_ONLY__
 #define OP(opassign, op)                                                       \
-  wi_element &operator opassign(const sycl::ext::oneapi::bfloat16 & rhs) {     \
+  wi_element &operator opassign(const sycl::ext::oneapi::bfloat16 &rhs) {      \
```

Comment:
Don't change the formatting of existing code.
Use clang-format --lines to avoid touching other parts of the code